web/js/components/context-menu/context-menu.js

```diff
@@ -74,10 +70,13 @@ function RightClickMenu(props) {
     };
   };
 
-
   useEffect(() => {
     events.on('map:singleclick', handleClick);
     events.on('map:contextmenu', handleContextEvent);
+    if (isCoordinateSearchActive) {
```

Comment:
It looks like this fixes the issue.  Alternatively, if the idea is that we don't need the listeners when `isCoordinateSearchActive` is `true`, we could skip registering them in the first place like:
```
useEffect(() => {
    if (isCoordinateSearchActive) return;
    events.on('map:singleclick', handleClick);
    events.on('map:contextmenu', handleContextEvent);
    return () => {
      events.off('map:singleclick', handleClick);
      events.off('map:contextmenu', handleContextEvent);
    };
  });
```

Reply:
Great suggestion! I think this is better solution and is a little more clear.  I'll put this change in and push it up.